CHANGELOG.md

```diff
@@ -1,5 +1,398 @@
+- Modified `renderApplication` to forward `initialProps` to `WrapperComponent` ([4f5a092bf6](https://github.com/facebook/react-native/commit/4f5a092bf68a0cd825328ce4a1e6bb41a8fad2e3) by [@rubennorte](https://github.com/rubennorte))
+- Differ: fix TinyMap to prevent possible crashes in `find()` and `begin()`, and prevent erased elements from being iterated over ([bb5d04366a](https://github.com/facebook/react-native/commit/bb5d04366ad66273fd908586a9f4bc7513b4276c) by [@JoshuaGross](https://github.com/JoshuaGross))
+- Add warning when scrollRef does not have a scrollTo method ([7f2515ece8](https://github.com/facebook/react-native/commit/7f2515ece8833f7a8adba025ef544013f89ae26f) by [@kacieb](https://github.com/kacieb))
+
+#### Android specific
+
+- Test infra: Robolectric 4.3.1 -> 4.4 upgrade ([d373a8d88c](https://github.com/facebook/react-native/commit/d373a8d88c30af910133d97ae973d256c4479929))
+- Added TurboModule example to RNTester when `USE_CODEGEN` is set ([94b198cfd5](https://github.com/facebook/react-native/commit/94b198cfd5d7d68a64bc8cb7b5a4df9f2be8b989) by [@fkgozali](https://github.com/fkgozali))
+- Add an explicit NDK version to Android template ([18ffe12203](https://github.com/facebook/react-native/commit/18ffe12203d03b4e960d61d7bb50cd02bba94663) by [@safaiyeh](https://github.com/safaiyeh))
+- Introduce JNativeModulePerfLogger ([c0dd11e532](https://github.com/facebook/react-native/commit/c0dd11e5326db15da2b84a21478eab1fc862a03b) by [@RSNara](https://github.com/RSNara))
+- `[Android] [Added] - Add support for shadowColor on API level >= 28` ([cfa4260598](https://github.com/facebook/react-native/commit/cfa42605989eee5a9de42bdb1259fb7f4d9451fb) by [@IjzerenHein](https://github.com/IjzerenHein))
+- Add accessibilityHint to TouchableNativeFeedback ([72285d808d](https://github.com/facebook/react-native/commit/72285d808dfce748287a19e2620d58517a5f76e7) by [@CMDadabo](https://github.com/CMDadabo))
+- Adds support for the `onProgress` event on `Image` ([fa0e6f8051](https://github.com/facebook/react-native/commit/fa0e6f8051d2208af467b789a2a9306ec7ddad76) by [@yungsters](https://github.com/yungsters))
+- Exposed getFlex method as part of ReactShadowNode API ([6570f7887b](https://github.com/facebook/react-native/commit/6570f7887b8824705ae09b5653d631428e17bc5f) by [@mdvacca](https://github.com/mdvacca))
+- Add \*.hprof files to gitignore ([69ce9c21d4](https://github.com/facebook/react-native/commit/69ce9c21d433a23ffb9934062b46fa64277ee255) by [@enesozturk](https://github.com/enesozturk))
+- Move DevSettingsActivity from main to debug ([d8e6c45782](https://github.com/facebook/react-native/commit/d8e6c45782a5c9132bb7ec315fe0b9ba3999e830) by [@invalid-email-address](https://github.com/invalid-email-address))
+
+#### iOS specific
+
+- Add disableButtonsIndices option to ActionSheetIOS component ([a7c1c5aff2](https://github.com/facebook/react-native/commit/a7c1c5aff24671bba609caeb82092a8de3d3b232) by [@lukewalczak](https://github.com/lukewalczak))
```

Comment:
There's a semi-duplicate item below which can be combined. Looks like it was caused by a revert.
```suggestion
- Add disableButtonsIndices option to ActionSheetIOS component ([a7c1c5aff2](https://github.com/facebook/react-native/commit/a7c1c5aff24671bba609caeb82092a8de3d3b232) and [f0bf4b0986](https://github.com/facebook/react-native/commit/f0bf4b09863f0426dd5f6753fb75a174c6c74cbd) by [@lukewalczak](https://github.com/lukewalczak))
```